flash_patcher/util/logging.py

```diff
@@ -0,0 +1,8 @@
+from logging import Formatter, StreamHandler, getLogger, INFO
+
+formatter = Formatter("%(levelname)s: %(message)s")
```

Comment:
Is there a reason we're declaring all these as global variables?

Ideally only the logger should be global, we should create a function here called `setupLogger()` and run it from patcher.py

Reply:
Fair point. Will do.